awx/ui/src/screens/Inventory/InventoryGroupDetail/InventoryGroupDetail.js

```diff
@@ -12,6 +11,7 @@ import { DetailList, Detail, UserDateDetail } from 'components/DetailList';
 import InventoryGroupsDeleteModal from '../shared/InventoryGroupsDeleteModal';
 
 function InventoryGroupDetail({ inventoryGroup }) {
+  const { inventoryType } = useParams();
```

Comment:
We call `useParams()` on line 25. We could use `params` from line 25 or we could pull inventory type from the `inventoryGroup.summary_fields.inventory.kind`.